.gitignore

```diff
@@ -9,3 +9,4 @@ extension/token.js
 node_modules
 docs
 lib/twilio/constants.js
+nodemon.json
```

Comment:
What's this for?

Reply:
I'm running nodemon locally to watch and auto build the SDK. I didn't want to commit my personal config.